### PR TITLE
Makes bee people capable of launching a fucking nuke head from they asses

### PR DESCRIPTION
**Pull request by DaddyIssues98**
## About The Pull Request
Make bee people use they stinger as projectiles

things that i have finished:
the ability
things that i need to do:
  balance
  sprites
  stinger surgery
  debuff to bee people if they dont have the stinger
  synthetic stinger so you can print more  incase you lose the original one
  put a video showing it working.
  maybe bees getting toxic damage every X time for no habing they stinger. 

## Why It's Good For The Game
  more unique mechanics to beepeople

## Changelog
:cl:
add: Added new things
add: Added more things
del: Removed old things
tweak: tweaked a few things
balance: rebalanced something
fix: fixed a few things
soundadd: added a new sound thingy
sounddel: removed an old sound thingy
imageadd: added some icons and images
imagedel: deleted some icons and images
spellcheck: fixed a few typos
code: changed some code
refactor: refactored some code
config: changed some config setting
admin: messed with admin stuff
server: something server ops should know
/:cl:

### CODE REVIEW

**Comment by DaddyIssues98** [comment]
i have 0 ideas of how to do the surgery, debugg mood and toxic damage over time code.

**Comment by JRDead** [comment]
nuke fart

File: code/modules/mob/living/carbon/human/species_types/beeperson.dm [hunk 1]
```diff
@@ -112,3 +127,36 @@
...
+		return
+
+	var/obj/item/organ/stinger
+	for(var/org in C.internal_organs)
+		if(istype(org, /obj/item/organ/stinger))
+			stinger = org
+			break
+
+	if(!stinger)
+		to_chat(C, "<span class='notice'>You don't have a tongue to shoot!</span>")
```
Comment (EgorDinamit) [inline code/modules/mob/living/carbon/human/species_types/beeperson.dm:156]: Copy-pasted code :)
Change "tongue" to stinger.

File: code/modules/surgery/advanced/reattachstingerbee.dm [hunk 2]
```diff
@@ -0,0 +1,31 @@
+/datum/surgery/advanced/reattach_stinger
+	name = "reattachment of stinger"
+	desc = "Suegery to reattach a bee stinger."
```
Comment (EgorDinamit) [inline code/modules/surgery/advanced/reattachstingerbee.dm:3]: Su**e**gery.

File: code/modules/surgery/surgery_step.dm [hunk 3]
```diff
@@ -176,6 +177,17 @@
...
+/datum/surgery_step/proc/get_obj_list()
+	if (!LAZYLEN(obj_needed))
+		return
+	var/list/objs = list()
+	for(var/R in obj_needed)
+		var/datum/obj/temp = GLOB.objs_needed_surgery[R]
+		if(temp)
+			var/objname = temp.name
+			objs += objname
+	return
```
Comment (EgorDinamit) [inline code/modules/surgery/surgery_step.dm:189]: You return nothing.
Don't you have to return something? Like, the list..?

File: code/modules/mob/living/carbon/human/species_types/beeperson.dm (hunk 1 above)
Comment (DaddyIssues98) [reply code/modules/mob/living/carbon/human/species_types/beeperson.dm:156]: i know. i just pushed without editing all, cause im lazy

File: code/modules/surgery/advanced/reattachstingerbee.dm (hunk 2 above)
Comment (DaddyIssues98) [reply code/modules/surgery/advanced/reattachstingerbee.dm:3]: ha

File: code/modules/surgery/surgery_step.dm (hunk 3 above)
Comment (DaddyIssues98) [reply code/modules/surgery/surgery_step.dm:189]: oh...right that is why probably it was getting a lot of errors, thanks

File: code/modules/surgery/organs/stinger.dm [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+/obj/item/hardened_stinger/embedded(atom/target)
+	if(isbodypart(target))
+		missed = FALSE
+
+/obj/item/hardened_stinger/unembedded()
+	var/turf/T = get_turf(src)
+	visible_message("<span class='warning'>[src] cracks and twists, changing shape!</span>")
+	for(var/i in contents)
+		var/obj/o = i
```
Comment (JohnFulpWillard) [inline code/modules/surgery/organs/stinger.dm:38]:
```suggestion
	for(var/obj/items as anything in contents)
```

Comment (JohnFulpWillard) [reply code/modules/surgery/organs/stinger.dm:38]: You'll have to replace the line under this if you use this

**Comment by DaddyIssues98** [comment]
readoing this in another PR

**Pull request closed without merging**